skimage/feature/tests/test_texture.py

```diff
@@ -228,6 +235,22 @@ def test_nri_uniform(self):
                         [ 9, 58,  0, 57,  7, 14]])
         np.testing.assert_array_almost_equal(lbp, ref)
 
+    def test_multiblock_lbp(self):
```

Comment:
Is there any reason for this to be in `TestGLCM` class ? I think it should be outside.